src/status_im/signing/core.cljs

```diff
@@ -151,35 +151,10 @@
                                                             custom-domain? %])
                                     nil))}])]
         (when-not in-progress?
-          (cond-> {:db (update db :signing/sign assoc :error nil :in-progress? true)}
-            (nil? action)                                 (assoc :signing/send-transaction-fx
-                                                                 {:tx-obj          tx-obj-to-send
-                                                                  :hashed-password hashed-password
-                                                                  :cb              cb})
-            (= action constants/ens-action-type-register) (assoc :json-rpc/call
-                                                                 [{:method "ens_register"
-                                                                   :params [chain-id tx-obj-to-send
-                                                                            hashed-password username
-                                                                            public-key]
-                                                                   :on-success
-                                                                   #(do
-                                                                      (cb (types/clj->json
-                                                                           {:result %}))
-                                                                      (watch-ens-tx-fn %))
-                                                                   :on-error #(cb (types/clj->json
-                                                                                   {:error %}))}])
-            (= action
-               constants/ens-action-type-set-pub-key)     (assoc :json-rpc/call
-                                                                 [{:method     "ens_setPubKey"
```

Comment:
we can't save ens name into backend db without `ens_register` or `ens_setPubKey` if i remember correctly?

Reply:
this was replaced by https://github.com/status-im/status-mobile/blob/86c5505c94f83e15242f07a3ad735ddcb8c6c4a3/src/status_im/ens/core.cljs#L199 and https://github.com/status-im/status-mobile/blob/86c5505c94f83e15242f07a3ad735ddcb8c6c4a3/src/status_im/ens/core.cljs#L134

Reply:
`ens_registerPrepareTx` can't replace `ens_register`, `ens_registerPrepareTx` only prepare transaction but won't save ens name into sqlite neither sync ens name across devices. `ens_setPubKeyPrepareTx` has similar issue.

`ens_registerPrepareTx` is more like `ens_prepareTxForRegisteringEnsUsername`, but we should use `ens_prepareTxForRegisteringEnsUsername` since it does more thing than `ens_registerPrepareTx` in the backend e.g. cache `txSignDetails` @flexsurfer

Reply:
thanks i'll take a look deeper then

Reply:
oh it seems like it wasn't a replacement, ok i confused it sorry, we still need ens_register, or use new methods from status-go, but need to check if it supports watch-ens-tx-fn